src/faldbt/project.py

```diff
@@ -467,6 +476,91 @@ def _lazy_setup_firestore(self):
                     "Could not find acceptable Default GCP Application credentials"
                 )
 
+    def airbyte_sync(
+        self,
+        config_name: str,
+        connection_id: str = None,
+        connection_name: str = None,
+        poll_interval: float = 10,
+        poll_timeout: float = None,
+        max_retries: int = 10,
+    ):
+        return self._run_el_sync(
+            config_name=config_name,
+            connection_key="connections",
+            connection_id=connection_id,
+            connection_name=connection_name,
+            poll_interval=poll_interval,
+            poll_timeout=poll_timeout,
+        )
+
+    def fivetran_sync(
+        self,
+        config_name: str,
+        connector_id: str = None,
+        connector_name: str = None,
+        poll_interval: float = 10,
+        poll_timeout: float = None,
+    ):
+        return self._run_el_sync(
+            config_name=config_name,
+            connection_key="connectors",
+            connection_id=connector_id,
+            connection_name=connector_name,
+            poll_interval=poll_interval,
+            poll_timeout=poll_timeout,
+        )
+
+    def _run_el_sync(
```

Comment:
if we are not using any private state from this class I think run_el_sync can be in its own class, you can keep `_el_configs` method for parsing here but take the `_run_el_sync` outside of this class..

I know similar things like firestore reverse etl is here but now that there are groups of tasks that are logically can be groups I say we do it seperate..